public/css/app.css

```diff
@@ -50,7 +50,7 @@ body {
   font-family: "Nunito Sans";
   width: 100%;
   height: 100vh;
-  background: url(/images/loginbg.jpg?f7edf72ec92d9ea89313bfd685a668d6);
+  background: #32c5e8;
```

Comment:
I see that you've changed the background. Are you aware of the error of the command "npm run watch" ? It keeps on looping because of the image stored on other directory. I am still finding ways on how to solve it. I've found one but it is not a good practice. Can you send me a screenshot of the new background of the login page?

Reply:
I'm aware that is  why i'm just using the command "npm run dev". I just changed the background to blue so it will be consistent to the registration page